php/whatsapp-with-vonage/src/index.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+require(__DIR__ . '/../vendor/autoload.php');
+require(__DIR__ . '/utils.php');
+
+use \Firebase\JWT\JWT;
+use \Firebase\JWT\Key;
+
+return function ($context) {
+    throw_if_missing($_ENV, [
+        'VONAGE_API_KEY',
+        'VONAGE_API_SECRET',
+        'VONAGE_API_SIGNATURE_SECRET',
+        'VONAGE_WHATSAPP_NUMBER',
+    ]);
+
+    if ($context->req->method === 'GET') {
+            return $context->res->send(get_static_file('index.html'), 200, [
+                'Content-Type' => 'text/html; charset=utf-8',
+            ]);
+        }
+
+    $authorizationHeader = isset($context->req->headers["authorization"]) ? $context->req->headers["authorization"] : "";
+    $jwtToken = explode(" ", $authorizationHeader)[1] ?? "";
+
+    $decoded = JWT::decode($jwtToken, new Key($_ENV['VONAGE_API_SIGNATURE_SECRET'], 'HS256'));
+    $decoded_array = (array) $decoded;
+
+    if (hash("sha256",$context->req->bodyRaw) !== $decoded_array["payload_hash"]) {
+        $context->res->json ([
+            'ok' => false,
+            'error' => "Payload Mismatch"
+        ], 401);
+    };
+
+    try {
+        throw_if_missing($context->req->body, ['from','text']);
+        } catch (\Exception $e) {
+            $context->res->json([
+                'ok'=> false,
+                'error'=> $e,
+            ], 400);
+        }   
```

Comment:
did the formatter format it like this?

Reply:
Ah yes, but i got this fixed by using another formatter